include/util.h

```diff
@@ -2,6 +2,7 @@
 #define _UTIL_H_
 
 #include <sys/types.h>
+#include <sys/termios.h>
```

Comment:
The original header file performs forward declaration and does not include `termios.h`. I think it's better not to pollute visible symbol space.